embed.fnc

```diff
@@ -2320,9 +2320,16 @@ Ep	|SV*	|get_re_gclass_nonbitmap_data				   \
 				|NULLOK SV **listsvp			   \
 				|NULLOK SV **lonly_utf8_locale		   \
 				|NULLOK SV **output_invlist
+#  endif
 #endif
-Ep	|void	|regprop	|NULLOK const regexp *prog|NN SV* sv|NN const regnode* o|NULLOK const regmatch_info *reginfo \
+#if defined(PERL_IN_REGCOMP_C) || defined(PERL_IN_REGEXEC_C) \
+ || defined(EXT_RE_RE_XS)
```

Comment:
Did you check to see if the existing defined(PERL_EXT_RE_BUILD) works?

Reply:
I kinda doubt that it does.

Reply:
I was unaware of it, I'll give it a go.

Reply:
Apparently it does work, I've dropped the extra symbol.